phpcs.xml

```diff
@@ -52,5 +52,9 @@
 	</rule>
 	<rule ref="WordPress.Files.FileName.InvalidClassFileName">
 		<exclude-pattern>includes/**/abstract-*.php</exclude-pattern>
+		<exclude-pattern>tests/*</exclude-pattern>
```

Comment:
We have been slowly fixing the names for test files. Can you maybe rename `unit-tests/api/system-status.php` instead of excluding all files inside `tests`?

Reply:
I'm absolutely on board with that, I wasn't aware that there was a movement to bring the test suite in-line with WordPress naming conventions.

Since I submitted this PR, it appears that [another PR has been merged that added a similar exclusion for the tests directory](https://github.com/woocommerce/woocommerce/pull/18551). When I look up the original author of the other PR...aww geez!

![It was me all along](https://memegenerator.net/img/instances/500x/66264771/it-was-me-all-along.jpg)

I'm happy to update this PR to rename the file and remove the potential merge conflict, if you'd prefer.

Reply:
Yes, I would prefer that so that we try to keep our test suite in-line with WordPress naming conventions. But before you go ahead and update this PR, let's hear if @mikejolley is on board since he merged the other PR and @claudiosanches as he added the PHPCS checks.

Thanks for your work and detailed explanations.

Reply:
File name is fine... But `print_r` should throw warnings since we could forget some while testing.
Not sure why you removed it, but there is also `wc_print_r()` that you can use instead.